### ISSUE TITLE
bug in printing blastn format

### ISSUE THREAD

**Issue by wrf** [lastz/lastz#20]
Hello,
When printing alignments in blastn format, the length is erroneously printed in the 10th column along with the second of the target coordinates.

```
scaffold_0001	scaffold_0001	71.37	227	65	0	107211	107437	107093	107319227	5.2e-73	271.6
scaffold_0001	scaffold_0001	62.60	246	92	0	107192	107437	107092	107337246	3.1e-63	239.2
scaffold_0001	scaffold_0001	74.92	327	82	0	107111	107437	107092	107418327	1.1e-116	416.6
```

For each line, the length of the match (4th column) is printed right after the target end, without a space or tab. This probably should not be there. In the first one, the target end should be 107319, not 107319227, three additional digits are added, which happens to be the length.

I see a (possibly) additional length variable on line 807 of output.c, should that be there?

Thanks
Warren

**Comment by rsharris** [comment]
This looks very much like issue #3. That was the result of a missing break in a switch case, and was fixed. (In the current repo, that's at line 1504 in genpaf.c).

There has been no update to the tagged release since that change. Could you verify that you have been using the 1.04.00 tagged release?  And if so, could you try cloning/building the current repo and verify that this corrects the problem?

If that's the case, I will try to make a new tagged release next week.

**Comment by wrf** [comment]
Yes, I was using the tagged release, not the source, output of --version is:
`lastz (version 1.04.00 released 20170312)
  score=int, sequence=31-bit, alloc=32-bit, allowBackToBackGaps=OFF
  built with gcc-5.4.0 "5.4.0 20160609"`

The current repo does not make this error.
Thanks

**Comment by rsharris** [comment]
Thanks for checking on that.  And thanks for reporting this.

I *do* want users to be using tagged releases, and I intended to make a tagged release with this fix shortly after the fix was made. Looks like it fell off the end of my task list though.

I'll try to make a new tagged release next week. The downside is there are some time consuming steps (updating version number, updating release notes, making sure the readme has the right version number, building a tarball for the other distribution channel, tagging the release in github, sending out a notification to the mailing list, etc. etc.). It can take a few hours.

In any case, thanks for reporting this, so that I will do all that.

**Comment by rsharris** [comment]
The new tagged release 1.04.03 contains the fix for this.
